zerver/views/realm_filters.py

```diff
@@ -41,3 +41,24 @@ def delete_filter(request: HttpRequest, user_profile: UserProfile, filter_id: in
     except RealmFilter.DoesNotExist:
         return json_error(_("Filter not found"))
     return json_success()
+
+
+@require_realm_admin
+@has_request_variables
+def update_filter(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    filter_id: int,
+    pattern: str = REQ(),
+    url_format_string: str = REQ(),
+) -> HttpResponse:
+    try:
+        do_update_realm_filter(
```

Comment:
We generally use `check_update_realm_filter` as the naming convention for functions where validation is done inside `actions.py`.  Though I notice this module isn't following that convention more generally, so it might be reasonable to defer to follow-up work.

Reply:
I have left this unchanged, since you mentioned that this will be taken in the follow-up work.